rmw_zenoh_cpp/src/detail/payload.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright 2024 Open Source Robotics Foundation, Inc.
```

Comment:
Can we move this definition to the existing `detail/zenoh_utils.hpp` header and have the implementation defined in `detail/zenoh_utils.cpp`?

Reply:
See https://github.com/ros2/rmw_zenoh/pull/343#issuecomment-2551797597.

Reply:
Nevermind.